tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -70,30 +70,36 @@ def convert_value(value, type_):
         #attempt numeric conversion
         try:
             if '.' in value:
-                return float(value)
+                return float(value.strip())
             else:
-                return int(value)
+                return int(value.strip())
         except ValueError as e:
             pass
         #bool
-        lval = value.lower()
+        lval = value.strip().lower()
```

Comment:
In which case do you expect the strings `true` and `false` to have leading / trailing spaces?